ada/ast.py

```diff
@@ -13962,6 +13962,38 @@ def family_type():
         """
         return Entity.spec.family_type.cast(TypeExpr)._.designated_type
 
+    @langkit_property(public=True, return_type=T.AcceptStmt.entity.array)
+    def accept_stmts():
+        """
+        Return an array of accept statements corresponding to this entry.
+        """
+        return Entity.find_accept_stmts(
+            # Find the body part of the task declaration for the current entry
+            Entity.parent_basic_decl.match(
+                lambda st=T.SingleTaskTypeDecl.entity: st.parent_basic_decl,
+                lambda other: other
```

Comment:
Since you already have that, what about adding an explicit case for `TaskTypeDecl` and raising a PropertyError for any other case?

Reply:
Sure, done!